<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="calc_input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="one_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimal_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changesign_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>(-)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mult_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minu_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
